r/R/dataset.R

```diff
@@ -72,6 +72,7 @@ open_dataset <- function(sources,
       } else {
         # Take the first one.
         schema <- sources[[1]]$schema
+        assert_is(schema, "Schema")
```

Comment:
This should go after L77, implicitly it's the "else" to `if (is.null(schema)) {` L68

Reply:
done